mode/type/standalone/repository/provider/jdbc/src/main/java/org/apache/shardingsphere/mode/repository/standalone/jdbc/sql/JDBCRepositorySQL.java

```diff
@@ -17,43 +17,41 @@
 
 package org.apache.shardingsphere.mode.repository.standalone.jdbc.sql;
 
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 import lombok.Getter;
 
-import javax.xml.bind.annotation.XmlAttribute;
-import javax.xml.bind.annotation.XmlElement;
-import javax.xml.bind.annotation.XmlRootElement;
-
 /**
  * JDBC repository SQL.
  */
-@XmlRootElement(name = "sql")
+@JacksonXmlRootElement(localName = "sql")
 @Getter
 public final class JDBCRepositorySQL {
     
-    @XmlAttribute(required = true)
+    @JacksonXmlProperty(isAttribute = true)
```

Comment:
We lost `required` here? Is there no such property with `JacksonXmlProperty`?

Reply:
- There is no direct equivalent.

 > Unfortunately XML is special: due to structure being what it is, empty Element looks either as "Object with no properties" or "empty String".

 - In https://github.com/FasterXML/jackson-dataformat-xml/issues/538, the Jaxkson maintainer has a different opinion on `required` than the JAXB API.  Should I use `@JsonProperty` to mark `required` and then set it as follows?
 ```java
 final XmlMapper xmlMapper = new XmlMapper();

 xmlMapper.configure(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES, true);
 ```

Reply:
- I'm not sure how the JAXB API handles the `required` attribute. 

- In my local unit test, `jaxb-runtime` does not throw an exception for empty elements, but gets null values normally like `jackson-dataformat-xml`. `required` seems to be just for IDE censorship.

Reply:
OK, I see. It could be better to open an issue since we need more investigation.

Reply:
> OK, I see. It could be better to open an issue since we need more investigation.

- I opened https://github.com/FasterXML/jackson-dataformat-xml/issues/625 . 
- A PR for testing is at #29577 .